crates/red_knot_python_semantic/src/types/class.rs

```diff
@@ -1102,9 +1108,12 @@ impl<'db> KnownClass {
             | Self::MethodWrapperType
             | Self::WrapperDescriptorType => KnownModule::Types,
             Self::NoneType => KnownModule::Typeshed,
-            Self::SpecialForm | Self::TypeVar | Self::StdlibAlias | Self::SupportsIndex => {
-                KnownModule::Typing
-            }
+            Self::SpecialForm
+            | Self::TypeVar
+            | Self::ParamSpec
+            | Self::TypeVarTuple
+            | Self::StdlibAlias
+            | Self::SupportsIndex => KnownModule::Typing,
             Self::TypeAliasType => KnownModule::TypingExtensions,
```

Comment:
you can fix the failing `known_class_doesnt_fallback_to_unknown_unexpectedly_on_low_python_version` test by looking up the symbol in the `typing_extensions` module rather than the `typing` module. The `typing` module doesn't have the `ParamSpec` and `TypeVarTuple` classes on low Python versions, but the `typing_extensions` module has it on all versions:

```suggestion
            Self::SpecialForm | Self::TypeVar | Self::StdlibAlias | Self::SupportsIndex => {
                KnownModule::Typing
            }
            Self::TypeAliasType | Self::TypeVar | Self::ParamSpec => KnownModule::TypingExtensions,
```

Reply:
Thank you, Is should've caught that sorry